plugins/rbac/src/components/CreateRole/RoleForm.tsx

```diff
@@ -81,6 +83,18 @@ export const RoleForm = ({
           if (roleName) {
             setToastMessage(`Role ${roleName} updated successfully`);
           } else {
+            values.permissionPoliciesRows.forEach(pp =>
+              pp.policies.forEach(async (policy: any) => {
+                if (policy.checked) {
+                  await rbacApi.createPolicy({
+                    entityReference: `role:${formik.values.namespace}/${formik.values.name}`,
+                    permission: `${pp.permission}`,
+                    policy: policy.label.toLowerCase(),
+                    effect: 'allow',
+                  });
+                }
+              }),
+            );
```

Comment:
We should notify the user if there's any error while creating the policies

Reply:
@debsmita1 yes. I am adding validations.